src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -135,6 +141,11 @@ export default {
             let displayMode = this.item.displayMode;
             return displayMode === 'all' || displayMode === 'value';
         },
+        unit() {
+            let value = this.item.value,
+                unit = this.metadata.value(value).unit;
+            return unit;
+        },
```

Comment:
Boom. We've come full circle. This is what I'd like the earlier hasUnits to look like. Nice work.